dist/jowl.spec

```diff
@@ -3,11 +3,16 @@ Version:        2.0.2
 Release:        1%{?dist}
 Summary:        CLI for JSON operations with Lodash
 
+%global gittag v%{version}
```

Comment:
Just drop `%gittag` and use `v%{version}` directly in `Source0`.

Reply:
Good point. Fixed.